deploy/Chart/templates/rp/deployment.yaml

```diff
@@ -69,6 +69,10 @@ spec:
         volumeMounts:
         - name: config-volume
           mountPath: /etc/config
+        {{- if eq .Values.global.aws.irsa.enabled true }}
+        - name: aws-iam-token
```

Comment:
```suggestion
        - name: aws-irsa-token
```

Reply:
as discussed offline keeping it `aws-iam-token`